tico/serialize/operators/op_max_pool2d_with_indices.py

```diff
@@ -72,6 +72,8 @@ def define_node(
         input = args.input
         kernel_size = args.kernel_size
         stride = args.stride
+        if not stride:
+            stride = kernel_size
```

Comment:
How about doing this in validate_args_kwargs? Defining proper `__post_init__` could do.